packages/circuit-ui/components/CurrencyInput/CurrencyInput.stories.tsx

```diff
@@ -33,7 +33,7 @@ const baseArgs = {
   currency: 'EUR',
   locale: 'de-DE',
   noMargin: true,
-  labelStyles: { maxWidth: '250px' },
```

Comment:
I think I added this as a fix back then because in a flex container with no set width, there would be a visual bug with the CurrencyInput. Can we check if the component renders properly without a max width with the new wrapper? (otherwise I'll add this to the list of bugs to address)

Reply:
This is a story file — removing the `max-width` doesn't appear to break anything in Storybook.